src/components/NotePage/NoteDetail/NoteDetail.tsx

```diff
@@ -470,7 +467,7 @@ export default class NoteDetail extends React.Component<
                     .reduce((prev, curr) => (
                       <>
                         {prev}
-                        <div className='separator'>></div>
+                        <div className='separator'>/</div>
```

Comment:
Let's also use `&frasl;` instead of `/`, will prevent some lint errors

Reply:
Fixed.